### ISSUE TITLE
botón subir en página web

### ISSUE THREAD

**Issue by marcoVillalobos** [marcoVillalobos/consultas-bootstrap#1]
No me aparece el boton una vez implementado en html en un servidor, solo aparece en localhost; tengo el estilo css y el script: aqui el codigo:
css:
.go-top { /* estilos del botón subir */

  position: fixed;
  bottom: 2em;
  right: 2em;
  text-decoration: none;
  color: #fff;
  background-color: rgba(0, 0, 0, 0.3);
  font-size: 12px;
  padding: 1em;
  display: none;
} 
.go-top:hover {
  background-color: rgba(0, 0, 0, 0.6);
}
html:
<a class="go-top" href="#"><img src="images/boton-arriba.png" alt="Placeholder image" width="40" class="img-responsive"></a></li>

script:
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

		  <script>

		    $(document).ready(function() {
		      // Show or hide the sticky footer button
		      $(window).scroll(function() {
		        if ($(this).scrollTop() > 200) {
		          $('.go-top').fadeIn(200);
		       } else {
		          $('.go-top').fadeOut(200);
		        }
		      });		 

		      // llamando a los estilos Css:
		      $('.go-top').click(function(event) {
		        event.preventDefault();
		        $('html, body').animate({scrollTop: 0}, 300);
		      })
		    });
		  </script>
